src/node_options.h

```diff
@@ -117,7 +117,7 @@ class EnvironmentOptions : public Options {
   std::vector<std::string> conditions;
   bool detect_module = true;
   bool print_required_tla = false;
-  bool require_module = false;
+  bool require_module = true;
```

Comment:
I think you need to update https://github.com/joyeecheung/node/blob/d79208a2cc62149bf5a96dbbce784d7173b9dac7/src/node_options.cc#L376-L379

(Add `true` at the end).

Reply:
After that, I expect a test to fail, because the documentation won't match the default.

Reply:
Thanks for catching it, fixed